app/controllers/questions_controller.rb

```diff
@@ -115,7 +115,7 @@ def update_as_suggested_edit
   end
 
   def destroy
-    unless check_your_privilege('Delete', @question, false)
+    unless check_your_privilege('flag_curate', @question, false)
       flash[:danger] = 'You must have the Delete privilege to delete questions.'
```

Comment:
Can you put the names of the privs in consts and use them in messages like these?  (a) there isn't a delete priv (though there's a priv that grants delete votes), so that might confuse users, but (b) instead of just hard-wiring it here, maybe put them all in one place so if we change the (UI) name of any of them we don't have to search the code for all uses?

Reply:
I think (correct me if I'm wrong, @luap42) these are internal names, i.e. constant - the public UI names are distinct from these.

Reply:
Yes this is true. However, the error messages for privilege errors will need to be improved/updated, too. It's on my TODO list, though. :)